### ISSUE TITLE
Sort widget tasks

### ISSUE THREAD

**Issue by ghost** [dmfs/opentasks#49]
Please allow sorting the tasks in the widget by priority and due date.

**Comment by GerdGrimm65** [comment]
YES! The ability to sort the widged by priority or date would be essential to me. Please add!
